src/services/balancer/pools/exits/handlers/exact-in-exit.handler.ts

```diff
@@ -167,4 +172,92 @@ export class ExactInExitHandler implements ExitPoolHandler {
 
     return amountsOut;
   }
+
+  private async getSdkPool(): Promise<PoolWithMethods | undefined> {
+    let sdkPool: PoolWithMethods | undefined;
+
+    try {
+      sdkPool = await getBalancerSDK().pools.find(this.pool.value.id);
+    } catch (error) {
+      captureBalancerException({
+        error,
+        context: {
+          tags: {
+            dependency: 'Balancer SDK',
+          },
+        },
+      });
+    }
+
+    return sdkPool;
+  }
+
+  private setLastExitRes(
+    sdkPool: PoolWithMethods,
+    {
+      evmBptIn,
+      exiter,
+      slippage,
+      shouldUnwrapNativeAsset,
+      singleTokenMaxOutAddress,
+    }: {
+      evmBptIn: string;
+      exiter: string;
+      slippage: string;
+      shouldUnwrapNativeAsset: boolean;
+      singleTokenMaxOutAddress?: string;
+    }
+  ) {
+    try {
+      this.lastExitRes = sdkPool.buildExitExactBPTIn(
+        exiter,
+        evmBptIn,
+        slippage,
+        shouldUnwrapNativeAsset,
+        // TODO: singleTokenMaxOutAddress address format. SDK fix?
+        singleTokenMaxOutAddress?.toLowerCase()
+      );
+    } catch (error) {
+      captureBalancerException({
+        error,
+        context: {
+          tags: {
+            dependency: 'Balancer SDK',
+          },
+        },
+      });
+    }
+  }
```

Comment:
I'm not sure we want to do this. We don't capture errors in the handlers, we throw and let the providers catch them so we can render an error for the user.